lesson2/Lesson2.py

```diff
@@ -0,0 +1,78 @@
+#1
+
+for i in range(11):
+    print(i**2)
+#2
+i = 0
+while 1 > 0:
```

Comment:
Forever cycle in python.
```suggestion
while True:
```